openni2_camera/launch/camera_only.launch.py

```diff
@@ -53,7 +53,7 @@ def generate_launch_description():
                     plugin='openni2_wrapper::OpenNI2Driver',
                     name='driver',
                     parameters=[{'depth_registration': True},
-                                {'use_device_time': False}],
+                                {'use_device_time': True}],
```

Comment:
this puts our example launch files more in line with ROS1 usage - where use_device_time was enabled by default